docs/commands/SetParameters.md

```diff
@@ -0,0 +1,99 @@
+# SetParameters
+
+Parameters setup command. The sensor confirms the execution of the command with [response](#response)
+Attention! Command is not fully documented!
+
+
+## Request
+
+### Request format
+
+> Size | Type     | Field
+> -----|----------|-------
+> 1    | byte     | command id = `0x03`
+> 1    | byte     | command dynamic size
+> 1    | byte     | [type](#type)
+> N    | bytes    | [data](#data)
+
+### Request parameters
+
+#### **type**
+
+Available parameters [types](#types)
+
+> Value | Description
+> ------|------------
+> 5     | [reporting data type](#reporting-data-type)
+
+#### **data**
+
+Type-specific structures.
+
+### Request examples
+
+Set parameter [reporting data type](#reporting-data-type) to "day"
+
+ Field           | Value    | Dump
+-----------------|----------|------
+ command id      | `3`      | `0x03`
+ command size    | `2`      | `0x02`
+ type            | `5`      | `0x05`
+ data            | `1`      | `0x01`
+
+
+## Response
+
+### Response format
+
+> Size | Type | Field
+> -----|------|-------
+> 1    | byte | command id = `0x03`
+> 1    | byte | length = `2`
+> 1    | byte | [type](#type)
+> 1    | byte | [status](#status)
+
+### Response parameters
+
+#### Status
+
+`1` - parameter setup was successful <br>
+`0` - parameter setting failed, parameter was not changed
+
+### Response examples
+
+success:
+
+ Field        | Value | Dump
+--------------|-------|------
+ command id   | `3`   | `0x03`
+ command size | `2`   | `0x02`
+ type         | `5`   | `0x05`
+ status       | `1`   | `0x01`
+
+failure:
+
+ Field        | Value | Dump
+--------------|-------|------
+ command id   | `3`   | `0x03`
+ command size | `2`   | `0x02`
+ type         | `5`   | `0x05`
+ status       | `1`   | `0x01`
+
+
+### Types
```

Comment:
it seems to be a general info (not a part of Response)

maybe it's best to put it on top?

or even better to put all parameters to a separate file as there are a lot of info
`/types.md` or `/parameter-types.md`

Reply:
added `/parameter-types.md`